docs/documenting-schema.rst

```diff
@@ -0,0 +1,73 @@
+.. _documenting-schema:
+
+Documenting schema
+=======================
+
+You can improve the experience of consuming your GraphQL API by documenting the types that it supports.
```

Comment:
Consuming? Sounds like we are slowly burning a GraphQL API fire pit.

```suggestion
You can improve the user experience of your GraphQL API by documenting the types that it supports.
```

Reply:
I stand by my phrasing here.  I think consumers and providers are fairly standard language in the API world.

To add to my defense, I'll note that the Apollo documentation uses almost identical language.

https://www.apollographql.com/docs/apollo-server/essentials/schema.html#comments

Reply:
Alright then!